src/Illuminate/Foundation/Auth/ResetsPasswords.php

```diff
@@ -24,8 +25,16 @@
      */
     public function showResetForm(Request $request, $token = null)
     {
+        $email = '';
+        if (is_null($token) == false) {
+            $email = DB::table('password_resets')
```

Comment:
I think this table is a config option? It cant be hard coded

Reply:
Your right, that is defined in the config/auth.php 
I'll update the pull request to use the config.

Feels like a stupid mistake